ios/.gitattributes

```diff
@@ -0,0 +1 @@
+*.pbxproj merge=union
```

Comment:
I saw some StackOverflow posts advising against using `merge=union`. Also when rebasing some of the refactor branches, I had to manually merge project files which would have failed with `merge=union` (particularly when files are renamed, reordered or deleted). So I'd prefer to be forced to manually review a few extra merges than have occasional bad merged.